### ISSUE TITLE
The wallet seed is generated incorrectly

### ISSUE THREAD

**Issue by tomkha** [Eligioo/nimiq-tools#5]
the wallet seed is composed of a private key followed by a public key:

https://github.com/nimiq/core-js/blob/master/src/main/generic/consensus/base/primitive/KeyPair.js#L55

while the import tool does the opposite:

https://github.com/nimiq-tools/nimiq-tools.github.io/blob/master/tools/import-from-mnemonic.html#L125

**Comment by sisou** [comment]
It's best to use the build-in functions here:

```diff
-$("#walletSeed").val(`${wallet.publicKey.toHex()}${wallet.keyPair.privateKey.toHex()}`)
+$("#walletSeed").val(`${Nimiq.BufferUtils.toHex(wallet.keyPair.serialize())}`)
```

Do you want to make a PR, @tomkha?

**Comment by tomkha** [comment]
@sisou you're welcome

**Comment by Eligioo** [comment]
Fixed in 581d0b7df40c8da1bed58d450e28d770205f08d9
